xbmc/addons/Addon.h

```diff
@@ -61,6 +61,14 @@ class CAddon : public IAddon
    */
   bool HasType(TYPE type) const override { return m_addonInfo->HasType(type); }
 
+  /**
+   * @brief To check complete addon (not only this) has a specific main type
```

Comment:
Can you clarify what a "main" type is here? I think it's the first extension point in addon.xml, right?

Reply:
yes, wasn't quite simple. main type means the first extension point including provided subcontent (like `ADDON_VIDEO` for instance). hope that is explained correctly and understandable. i would add it to the doxy